inotify/adapters.py

```diff
@@ -231,6 +231,10 @@ def event_gen(
                         in self._handle_inotify_event(fd):
                     last_hit_s = time.time()
 
+                    if handle_in_ignored and\
```

Comment:
Just make it default behavior. The user shouldn't have a choice. Reword the commit message to mention that we're specifically handling the IN_IGNORED event, now.